packages/twenty-server/src/integrations/tracing/useTracing.ts

```diff
@@ -0,0 +1,31 @@
+import * as Sentry from '@sentry/node';
+import { handleStreamOrSingleExecutionResult, Plugin } from '@envelop/core';
+import { OperationDefinitionNode, Kind } from 'graphql';
+
+export const useTracing = (): Plugin => {
+  return {
+    onExecute({ args }) {
+      console.log('onExecute');
+      console.log(args);
+      const transactionName = args.operationName || 'Anonymous Operation';
+      const rootOperation = args.document.definitions.find(
+        (o) => o.kind === Kind.OPERATION_DEFINITION,
+      ) as OperationDefinitionNode;
+      const operationType = rootOperation.operation;
+
+      if (Sentry.isInitialized()) {
+        Sentry.setTags({
+          operationName: transactionName,
+          operation: operationType,
+        });
+        Sentry.getCurrentScope().setTransactionName(transactionName);
```

Comment:
marked as deprecated but still works and used by most packages and might be un-deprecated soon? https://github.com/getsentry/sentry-javascript/issues/10846

Reply:
I've spent enough time on this so I'd say we go with this, not a big debt if we need to update it later! And it looks bad when we let sentry figure out the transaction name (everything under POST /graphql)